.cypress/integration/2_notebooks.spec.js

```diff
@@ -40,7 +42,7 @@ const moveToTestNotebook = () => {
     .click();
 };
 
-describe('Adding sample data and visualization', () => {
+describe.only('Adding sample data and visualization', () => {
```

Comment:
Is `.only` supposed to be here? Went to research it and at a glance it looks like a debugging tool